css/app.css

```diff
@@ -39,7 +39,7 @@ body {
     height: 100%;
     width: 100%;
     z-index: -10;
-    margin-left:400px;
+    /*margin-left:400px;*/
```

Comment:
Why was this necessary?  Removing this seems to have necessitated all the 400s below for individual child tags.